### ISSUE TITLE
Adding to the Readme also what are the original requirements for becoming a Paladin in QFG3

### ISSUE THREAD

**Issue by lwcorp** [AshLancer/QFG3-Fan-Patch#2]
The Readme has a section of "What are the exact **new** requirements for becomming a Paladin in QFG3?"

Can you:
1. Fix "becomming" to "becoming"?
1. Add another question first of _"What are the exact **original** requirements for becomming a Paladin in QFG3?"_
1. Change the "new requirements" to only your additions, like "the new requirements are the original ones plus not doing...", then a bullet list.

**Comment by AshLancer** [comment]
Yeah, thanks for catching that's a common typo I tend to make. Will fix!

It's a good idea, I'll re-list the requirements in the next release.
